### ISSUE TITLE
react-native-gallery and react-native-lock-android

### ISSUE THREAD

**Issue by heri** [reindexio/reindex-examples#4]
I can't make the `react-native-gallery` example work. It seems stuck in the auth0 callback. 

I looked and saw [react-native-lock-android](https://github.com/auth0/react-native-lock-android) was deprecated. Apparently you have to use https://github.com/auth0/react-native-lock now 

Unless someone manages to run the example with `react-native-lock-android`?

**Comment by freiksenet** [comment]
Hi!

Sorry for the late answer.

Do you have the Auth0 Config set up? You can read the logging with adb on Android, maybe this will help.

I'll look into new lock versions this week and update accordingly.

**Comment by freiksenet** [comment]
There is some weird problem with how the bluebird fromCallback got imported and that caused login to infinitely loop. I've fixed that, but now there seem to be some problems in other parts of the application, maybe it's newer React Native version. I'll investigate further.

**Comment by freiksenet** [comment]
OK, I understand it now. There are some incompatibilities with new Babel version. I've fixed Babel version and it works fine without code changes. @heri, tell me if it works for you now. Remember to reinstall node modules and clean React Native build cache.

**Comment by heri** [comment]
Hi

I still have the infinite loop with fromCallback, after reinstalling node_modules and deleting app/build cache

I might miss a command or a file that hasn't been committed?

Thanks for looking into thi

**Comment by freiksenet** [comment]
Do you see anything in the log? On android you can

```
adb logcat *:S ReactNative:V ReactNativeJS:V
```

**Comment by heri** [comment]
I updated node from `4.2.2` to `6.1.0` and now it works.

It is very slow (takes 1 minute to switch to AddPicture component, another 1 minute to take a picture, 420 frames dropped). I don't know if it's `react-native-camera`, Reindex or GenyMotion. I'll need to try with a physical device if it's faster.

I also have error when trying to save a picture `Possible Unhandled Promise Rejection (id: 1): filePath.split is not a function`

I have played with the provided queries and mutations on the GraphiQL myreindex, it's nice. I think what we will do is we will use reindex for a test project

Thanks !
